javascript/linkedList/linkedList/node.js

```diff
@@ -1,7 +1,7 @@
-'use strict';
```

Comment:
Prettier made these changes automatically. I have my editor set up to apply Pretter on save sooooooo this isn't just me nitpicking. My tool did it!